### PR TITLE
Updated Modal Close

### PR DESCRIPTION
**Pull request by arse0039**
Updated about page to allow the modal window to be closed when a user clicks anywhere on the screen while the modal window is open.
Removed exit button from modal window.
Minor tweaks to formatting and fixed typo.

### CODE REVIEW

File: src/components/aboutModal.tsx [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
+  return (
+    <div
+      className={`modal-wrapper ${isExpanded ? "expanded" : ""}`}
+      onClick={handleExitButtonClick}
+    >
+      <div
+        className={`expandingDiv ${isExpanded ? "expanded" : ""}`}
+        ref={expandingDivRef}
+      >
+        {/* <button className="exit-button" onClick={handleExitButtonClick}>
```
Comment (arse0039) [inline src/components/aboutModal.tsx:47]: Kept this in for reference, just in case something breaks

File: src/constants/data.tsx [hunk 2]
```diff
@@ -1,169 +1,181 @@
...
     I often had the opportunity to collaborate with software engineering teams. I always had an appreciation \
     for their contribution to the projects we worked on and thought it would be interesting to be on that side of the work,\
     actually ‘building’ the ideas that the business had come up with. Two years ago I started actively working \
     towards making the switch to a career as a Software Engineer by returning to school to pursue a degree in Computer \
-    Science at Oregon State University.", "Over the course of my degree program, I worked full-time, held a part-time position as a teaching \
+    Science at Oregon State University.",
+    "Over the course of my degree program, I worked full-time, held a part-time position as a teaching \
     assistant, and last summer I had a successful software engineering internship where I got professional experience working on a full-stack \
-    project. I returned to school following the intership and graduated in December.", "In addition to working on projects with Brandon, I'm now \
+    project. I returned to school following the internship and graduated in December.",
```
Comment (arse0039) [inline src/constants/data.tsx:123]: Prettier formatted a lot of things but all I did was fix a type here.

Comment (davidclaphan) [reply src/constants/data.tsx:123]: appreciate it!

**Comment by davidclaphan** [comment]
looks great
no issues with scrolling or loading correctly for either profile

**Review by davidclaphan — APPROVED** [review]

**Pull request merged**